redirects.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "source":"/lib/ssr/ssr/q/platform/js/",
-    "target":"/javascript/prev/build-a-backend/sever-side-rendering/",
+    "target":"/javascript/prev/build-a-backend/server-side-rendering/",
```

Comment:
fixed the spelling and also updated the folder and directory.mjs to reflect the change